src/tacacs/nss/patch/0012-Write-error-message-to-syslog-when-add-user-failed.patch

```diff
@@ -0,0 +1,40 @@
+From 3f88c7fe559a004b580fd515f27a1cb3e06a32f9 Mon Sep 17 00:00:00 2001
+From: liuh-80 <liuh@microsoft.com>
+Date: Wed, 23 Aug 2023 11:18:45 +0800
+Subject: [PATCH] Write error message to syslog when add user failed or connect to TACACS server failed.
+
+---
+ nss_tacplus.c | 10 ++++------
+ 1 file changed, 4 insertions(+), 6 deletions(-)
+
+diff --git a/nss_tacplus.c b/nss_tacplus.c
+index c22d6bd..f2ba035 100644
+--- a/nss_tacplus.c
++++ b/nss_tacplus.c
+@@ -540,8 +540,7 @@ static int create_or_modify_local_user(const char *name, int level, bool existin
+             snprintf(sgid, 10, "%d", user->gid);
+             snprintf(home, 63, "/home/%s", name);
+             if(0 != user_mod_add(command, name, sgid, user->secondary_grp, user->info, home, user->shell)) {
```

Comment:
better to change existing patch file `0011-Replace-popen-shell-execution-with-safer-execle.patch`

Reply:
and also fix the indentation for next 3 lines.

Reply:
Fixed.